src/main/java/pro/sky/courseworktelegrambot/services/AdoptionService.java

```diff
@@ -0,0 +1,243 @@
+package pro.sky.courseworktelegrambot.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import javax.persistence.EntityNotFoundException;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Service;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import pro.sky.courseworktelegrambot.entities.*;
+import pro.sky.courseworktelegrambot.exceptions.ShelterNotFoundException;
+import pro.sky.courseworktelegrambot.exceptions.TelegramException;
+import pro.sky.courseworktelegrambot.exceptions.UserOrPetIsBusyException;
+import pro.sky.courseworktelegrambot.repositories.*;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.Collection;
+import java.util.List;
+
+@Service
+public class AdoptionService {
+    private static final Logger logger = LoggerFactory.getLogger(AdoptionService.class);
```

Comment:
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(AdoptionService.class);
```
static И final в верхнем регистре